test/clj_kondo/single_arity_comparison_test.clj

```diff
@@ -0,0 +1,42 @@
+(ns clj-kondo.single-arity-comparison-test
+  (:require
+   [clj-kondo.test-utils :refer [lint! assert-submaps]]
+   [clojure.test :as t :refer [deftest is testing]]))
+
+(deftest single-arity-comparison-test
+  (testing "test full linting error for single arity comparison in clojure"
+    (assert-submaps
+     '({:file "<stdin>", :row 1, :col 1, :level :warning,
+        :message "single arity use of clojure.core/= is constantly true"})
+     (lint! "(= 1)")))
+
+  (testing "test full linting error for single arity comparison in cljs"
+    (assert-submaps
+     '({:file "<stdin>", :row 1, :col 1, :level :warning,
+        :message "single arity use of cljs.core/not= is constantly false"})
+     (lint! "(not= 1)" "--lang" "cljs")))
+
+  (testing "test linting comparison operators with single arity"
+    (doseq [lang ["clj" "cljs"]
+            op ["=" ">" "<" ">=" "<=" "=="]
+            :let [errors (lint! (str "(" op " 1)") "--lang" lang)]]
+      (is (= (count errors) 1))
```

Comment:
`(count errors) 1` -> `1 (count errors)`

clojure.test expects `expected actual` order; else error reports are affected

Reply:
Thanks! Fixed it